aci/resource_aci_l3extvirtuallifp.go

```diff
@@ -246,6 +250,43 @@ func setVirtualLogicalInterfaceProfileAttributes(l3extVirtualLIfP *models.Virtua
 	return d, nil
 }
 
+func getL3extRsVSwitchEnhancedLagPol(client *client.Client, dn, tDn string) string {
+	l3extVirtualLIfPLagPolAttDn := fmt.Sprintf("%s/rsdynPathAtt-[%s]/vlifplagpolatt", dn, tDn)
+	log.Printf("[DEBUG] l3extVirtualLIfPLagPolAttDn %v", l3extVirtualLIfPLagPolAttDn)
+	l3extRsVSwitchEnhancedLagPol, err := client.ReadRelationl3extRsVSwitchEnhancedLagPol(l3extVirtualLIfPLagPolAttDn)
+	if err != nil {
+		log.Printf("[DEBUG] Error while reading relation l3extRsVSwitchEnhancedLagPol %v", l3extRsVSwitchEnhancedLagPol)
+		return ""
+	} else {
+		return l3extRsVSwitchEnhancedLagPol.(string)
+	}
+}
+
+func getAndSetL3extRsDynPathAttFromLogicalInterfaceProfile(client *client.Client, dn string, d *schema.ResourceData) (*schema.ResourceData, error) {
+	l3extRsDynPathAttData, err := client.ReadRelationl3extRsDynPathAttFromLogicalInterfaceProfile(dn)
+	if err != nil {
+		log.Printf("[DEBUG] Error while reading relation l3extRsDynPathAtt %v", err)
+		d.Set("relation_l3ext_rs_dyn_path_att", make([]interface{}, 0, 1))
+		return nil, err
+	} else {
+		l3extRsDynPathAttMap := l3extRsDynPathAttData.([]map[string]string)
+		st := make([]map[string]string, 0)
```

Comment:
You can move the st before the if condition and use l3_ext_rs_dyn_path or some good name instead st.

Reply:
ok